Sources/Action/Action.swift

```diff
@@ -61,13 +61,19 @@ public final class Action<Input, Element> {
 
         let enabledSubject = BehaviorSubject<Bool>(value: false)
         enabled = enabledSubject.asObservable()
+
+        let errorsSubject = PublishSubject<ActionError>()
+        errors = errorsSubject.asObservable()
         
         executionObservables = inputs
             .withLatestFrom(enabled) { $0 }
             .flatMap { input, enabled -> Observable<Observable<Element>> in
                 if enabled {
-                    return Observable.of(workFactory(input).shareReplay(1))
+                    return Observable.of(workFactory(input)
+                                             .do(onError: { errorsSubject.onNext(.underlyingError($0)) })
+                                             .shareReplay(1))
```

Comment:
nit: Introducing variable for the execution observable makes this part more readable.

```swift
let execution = workFactory(input)
    .do(onError: { errorsSubject.onNext(.underlyingError($0)) })
    .shareReplay(1)

return Observable.of(execution)
```